fix 24: updated button,field, set half page fixed

bhanu2004 opened the pull request:
fixes #24

1. updated the styling of input fields for dark mode
2. updated the button
3. made the right side of page fixed, scrollable and set height to full screen
![image](https://github.com/swciitg/PortfolioBuilder/assets/75109116/7b65a0e2-c4e1-4fac-a1ad-8f1735606e39)
![image](https://github.com/swciitg/PortfolioBuilder/assets/75109116/89b44693-d896-4e3b-808e-79f94e509192)

AnkitGurwan commented:
@bhanu2004 
When we are opening preview, we are getting three scrolls, kindly make it two (**similar to code**).
Also, if possible, make the navbar fixed.
Do let me know if you face any issue.

bhanu2004 commented:
@AnkitGurwan 
i have fixed the three scroll issue
also made navbar fixed
and modified the height of right side since some of the content wasn't visible

AnkitGurwan commented:
Thanks for contributing!!
